### PR TITLE
Replace tentative proposal-style wording

### PR DESCRIPTION
**Pull request by aaaidan**
Strengthen wording to reflect the firm industry support SemVer has gained since the spec was first written.

To address #778

Comments and contributions welcome.

### CODE REVIEW

File: semver.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ specified too loosely, you will inevitably be bitten by version promiscuity
 Dependency hell is where you are when version lock and/or version promiscuity
 prevent you from easily and safely moving your project forward.
 
-As a solution to this problem, we propose a simple set of rules and
+SemVer addresses this with a simple set of rules and
 requirements that dictate how version numbers are assigned and incremented.
 These rules are based on but not necessarily limited to pre-existing
```
Comment (jwdonahue) [inline semver.md:36]: I think there should be a comma here, while you're at it.  Should be "...based on, but..."

**Review by jwdonahue — APPROVED** [review]

**Comment by jwdonahue** [comment]
I like this. I wouldn't call it "strengthening", but they are good changes. Industry support isn't relevant either. The fact is that semantic versioning, in its many forms, is part of the technical lexicon. SemVer as presented here, is one form of semantic versioning, but the proposal here is limited to just this one scheme. I would reword this to indicate that this is one, arguably the most popular, form of semantic versioning (possibly the first scheme to be formally named as such?).

**Comment by aaaidan** [comment]
I just had a look, and I can’t find any evidence of anyone using the phrase “semantic versioning” to describe their non-SemVer versioning strategy. 

Perhaps it’s just buried, for me, by the common usage that “semantic versioning” == SemVer. In itself, I wonder if that indicates we needn’t worry about falsely “claiming” that phrase. 

Realistically, I don’t think it would be reasonable in 2021 and beyond to use the term “semantic versioning” in software circles and not expect the listener to think you’re talking about SemVer. 

If I’m right, I think it’s okay to use these terms more or less interchangeably. 

Thoughts?

**Comment by jwdonahue** [comment]
@aaaidan, WikiPedia mentions several forms of semantic versioning: https://en.wikipedia.org/wiki/Software_versioning#Semantic_versioning

There's always been two kinds of versioning: semantic and numerical, both have histories going back thousands of years.

The "semantic X" craze in the software developers community, was kick started at least as far back as May 2001, in a Scientific American article co-authored by Tim Berners-Lee, James Handler and Ora Lassila titled "The Semantic Web", but they merely defined how **_semantics_**, a field as old as language studies, could be applied to the web, such that machines could process it without having to parse human language. 

According to the Google: Tom Preston-Werner (aka Mojombo) used the phrase "semantic versioning" when he created SemVer back in 2013, but I recall tracing it back to the 2008..2010 range a few years ago, still attributed to Mojombo, but his original repo is no longer available on GitHub.  I think his main contribution was the marriage of a **_common_** syntax **_and_** semantics, with an easy to remember brand "SemVer", at a time when there were a lot of newbies coming on the scene and chaotically reinventing every wheel they could think of.

The history of software versioning is littered with a huge range of semantics and syntaxes, many of which originated in the phenolic board labeling schemes used by the electronics industry, which itself was adopted from common book and journal versioning schemes that date back to the first printing presses. 

SemVer may dominate in the NPM, Nuget, and a handful of other silos, but there are many popular software libraries, operating systems and apps that use other forms of semantic versioning, some of which have more specific labels on them in the [WikiPedia](https://en.wikipedia.org/wiki/Software_versioning) article.

**Comment by aaaidan** [comment]
Thanks for the history, very interesting. 

On reflection, the “this is” sentence risks implying SemVer is the only semantic versioning. As you explain, that’s not true. It’s also doesn’t seem to be critically important, and only reinforces the “we propose” vibe I’m trying to update.

**Comment by aaaidan** [comment]
Not sure whether this PR needs more time for discussion and review. Maybe it can it be merged as is? 
@jwdonahue @JohnTitor 

(I don’t know how the PR lifecycle works for the semver repo. )

**Comment by jwdonahue** [comment]
@aaaidan, while I've had a lot to say about many of the PR's around here, I do not have any control over whether they are accepted. See the [Contributor's Guide](https://github.com/semver/semver/blob/master/CONTRIBUTING.md)

**Review by Jpac7-ai — COMMENTED** [review]
Change it to a normal place

**Comment by aaaidan** [comment]
Moderators, I think @Jpac7-ai could be a bot, as it’s a new account and has had some posts closed with the spam label.
